javascript/javascript2/week2/hyf-bay/main.js

```diff
@@ -0,0 +1,82 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+let resultedProducts = products;
+
+console.log(productsUl);
+
+function renderProducts(products) {
+  products.forEach(product => {
+    const li = document.createElement('li');
+    let shipsToHTML = '';
+    product.shipsTo.forEach(country => shipsToHTML += `<li>${country}</li>`);
+    li.innerHTML = `
+            <ul>
+                <li>${product.name}</li>
+                <li>${product.price}</li>
+                <li>${product.rating}</li>
+                <ul class="ships-to">${shipsToHTML}</ul>
```

Comment:
Element `ul` not allowed as child of element `ul` . You must wrap every inner `ULs` with an `LI`